From 2acc8b24b8291fccb54fb8d8057efc2e69b16851 Mon Sep 17 00:00:00 2001
From: oliverwen <oliverwen@google.com>
Date: Wed, 29 Jun 2016 13:19:26 -0700
Subject: [PATCH] CHROMIUM: ath10k Set default antenna to 0x0 for 2G

TEST=Run OTA performance test at HT20/2G

BUG=chrome-os-partner:51964
CQ-DEPEND=CL:341528

Signed-off-by: oliverwen <oliverwen@google.com>
(cherry picked from commit aca77a1dde0d90bd202d0b7e007ccc126799685a)
(cherry picked from wireless-3.18)

Change-Id: Ic13543825f1b4f31ac7b4679fc907e98ac0a5ff9
Reviewed-on: https://chromium-review.googlesource.com/341529
Commit-Ready: Yixiang Li <yixiang@google.com>
Tested-by: Oliver Wen <oliverwen@chromium.org>
Reviewed-by: Yixiang Li <yixiang@google.com>
---
 drivers/net/wireless/ath/ath10k/wmi.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath10k/wmi.h b/drivers/net/wireless/ath/ath10k/wmi.h
index f820904a5c3..955fd1f4a1b 100644
--- a/drivers/net/wireless/ath/ath10k/wmi.h
+++ b/drivers/net/wireless/ath/ath10k/wmi.h
@@ -7455,7 +7455,7 @@ struct wmi_chan_survey_req_cmd {
 #define WMI_SMART_ANT_DISABLED_ANT_SEL	2
 #define WMI_SMART_ANT_DISABLED_MODE_PARALLEL	1
 #define WMI_SMART_ANT_GPIO_MAX		4
-#define ATH10K_DEFAULT_ANTENNA_2G 2
+#define ATH10K_DEFAULT_ANTENNA_2G 0
 #define ATH10K_DEFAULT_ANTENNA_5G 6
 
 struct wmi_pdev_set_smart_ant_cmd {
-- 
2.43.0

